### ISSUE TITLE
....

### ISSUE THREAD

**Issue by RagaSonic** [catgirlfighter/nekopaw#388]
.

**Comment by JarieSuisen** [comment]
? Are you taking up the project for updates?

**Comment by RagaSonic** [comment]
No, I am not a specialist in coding. But as the project is abandoned, I sought a solution by myself. (This has fixed my problem, so I've provided a solution for other users who may not know how it works.) For my part, I do not intend to resume the project. But when the program crashes I try to find a solution.
